src/Microsoft.Sbom.Common/ComplianceStandard/NoneComplianceStandardEnforcer.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text.Json;
+using Microsoft.Sbom.Contracts.Enums;
+using Microsoft.Sbom.JsonAsynchronousNodeKit.Exceptions;
+using Microsoft.Sbom.Parsers.Spdx30SbomParser.ComplianceStandard.Interfaces;
+
+namespace Microsoft.Sbom.Common.ComplianceStandard;
+
+public class NoneComplianceStandardEnforcer : IComplianceStandardEnforcer
+{
+    public ComplianceStandardType ComplianceStandard => ComplianceStandardType.None;
+
+    public NoneComplianceStandardEnforcer() { }
+
+    public string GetComplianceStandardEntityType(string entityType)
+    {
+        return ComplianceExtensions.GetCommonEntityType(entityType);
```

Comment:
Use extension method:
```suggestion
        return entityType.GetCommonEntityType();
```